varia/style-editor.css

```diff
@@ -727,6 +727,14 @@ object {
 	margin-bottom: 0;
 }
 
+.wp-block-group {
+	/* Treating H2 separately to account for legacy /core styles */
+}
```

Comment:
Are we getting this empty rule because of the comment?

Reply:
seems like it. interestingly switching to a `//` comment removes it...

Reply:
Yeah, I think the Sass compiler completely removes `//` but it keeps `/* … */`. 

_I believe_ there’s a separate, later script on wpcom that minifies each theme stylesheet which removes all code comments in CSS.